www/styles/admin-swat-local.css

```diff
@@ -176,28 +304,45 @@ span.swat-checkbox-wrapper {
 span.swat-checkbox-shim {
 	-webkit-appearance: none;
 	-moz-appearance: none;
+	z-index: 2;
 	display: inline-block;
-	width: 18px;
-	height: 18px;
+	width: 16px;
+	height: 16px;
 	vertical-align: middle;
 	margin: 3px 3px 3px 4px;
 	text-indent: -25px;
-	border: none;
+	border: 0;
 	padding: 0;
 	background: url(../images/admin-checkbox-radio-buttons.png) no-repeat;
```

Comment:
In a closed system like ours where the vast majority of users are using Chrome/Safari (mostly on OS X, it seems), I'm not too concerned about the OS/browser variability. It's also important to consider consistency with forms in all other sites/systems the user visits (though that is getting more custom by default).

Maintaining custom styles like this seems overkill for a system like this, but since the work is already done in this case, we're probably fine.